tests/PHPUnit/Integration/Http/Post.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+// used in integration tests to see if POST method works.
+// for security reasons we allow max 3 post vars, each key and value is only allowed to have max 6 hex characters
+
+function accept($key)
```

Comment:
If someone installs Piwik via Git on production this file will be on a production server. Therefore I added a little check to make sure people can't do too many things with it